gcloud/_helpers.py

```diff
@@ -238,3 +241,35 @@ def __init__(self, project=None, implicit=False):
 
 
 _DEFAULTS = _DefaultsContainer(implicit=True)
+
+
+class _ClientProxy(object):
```

Comment:
I strongly prefer we avoid Python-fu, especially when it comes to custom `__getattr__`. Is this really necessary?